packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -259,28 +259,35 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onCreateLocalStreamView && onCreateLocalStreamView(localVideoViewOptions);
     }
     return (
-      <VideoTile
-        key={localParticipant.userId}
-        userId={localParticipant.userId}
-        renderElement={
-          localVideoStream?.renderElement ? (
-            <>
-              {
-                /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-                localCameraCycleButton(localVideoCameraCycleButtonProps)
-              }
-              <StreamMedia videoStreamElement={localVideoStream.renderElement} />
-            </>
-          ) : undefined
-        }
-        showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={isNarrow ? '' : strings.localVideoLabel}
-        initialsName={localParticipant.displayName}
-        styles={localVideoTileStylesThemed}
-        onRenderPlaceholder={onRenderAvatar}
-        isMuted={localParticipant.isMuted}
-        showMuteIndicator={showMuteIndicator}
-      />
+      <Stack
+        tabIndex={0}
+        aria-label={'Tile keyboard movement Dropdown Hidden'}
+        aria-role={'drop down'}
```

Comment:
drop down doesn't highlight the whole role of this widget. It has a hidden drop down to allow moving the PiP but the PiP itself also shows useful information beyond the drop down.
Consider `diaglog` as the role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role

The something in the aria-description should mention about the available dropdown to move the dialog about